### ISSUE TITLE
Some requests :d

### ISSUE THREAD

**Issue by VukBakic** [haxball/haxball-issues#371]
It would be nice if we could get access to
player.flag
player.avatar
player.ping
Also would be nice if we could edit player color (admin have yellow name color so should not be hard to make :d)
something like onPlayerTryingToJoin (orPlayerConnecting) would be good.
Also more map editing features would be really good :d

**Comment by VukBakic** [comment]
You don't need to use it if you don't want.

**Comment by Leo-MapCreator** [comment]
I agree with the map part, being able to create retangles that works like disc's is a really good idea, or even, lineWidth would be good also
